README.md

```diff
@@ -86,6 +86,14 @@ respond_to do |format|
 end
 ```
 
+### Specify a custom header row
+
+```ruby
+respond_to do |format|
+  format.csv  { render csv: @locations, header: 'Address', 'Zip Code', only: [:address, :zip] }
```

Comment:
Does the `header` option need to be an array? It doesn't seem like this example is valid.